Python/legacy_tracing.c

```diff
@@ -59,6 +59,16 @@ static PyObject *
 sys_profile_func3(
     _PyLegacyEventHandler *self, PyObject *const *args,
     size_t nargsf, PyObject *kwnames
+) {
+    assert(kwnames == NULL);
+    assert(PyVectorcall_NARGS(nargsf) == 3);
+    return call_profile_func(self, Py_None);
+}
+
+static PyObject *
+sys_profile_func3_args2(
```

Comment:
Since this is only used for `PyTrace_RETURN` events, maybe rename it to `sys_profile_return`?